### PR TITLE
Task table

### PR DESCRIPTION
**Pull request by jon-wehner**
Added table with task info on main page and front end javascript to serve new rows via API calls.

### CODE REVIEW

File: public/javascripts/group.js [hunk 1]
```diff
@@ -57,9 +57,10 @@ const postForm = async (url, formData) => {
 
 addGroupForm.addEventListener("submit", async (e) => {
   const formData = new FormData(addGroupForm);
-  e.preventDefault();
+  // e.preventDefault();
```
Comment (edherm) [inline public/javascripts/group.js]: If this is not in use, let's remove this line of code.

File: public/javascripts/task.js [hunk 2]
```diff
@@ -100,25 +87,32 @@ const postForm = async (url, formData, httpMethod) => {
...
-  return listGroup;
+const fillTableCell = (data) => {
+  const td = document.createElement("td");
+  const p = document.createElement("p");
+  p.innerText = data;
+  td.append(p);
+  return td
+}
+
+const createTableRow = (task) => {
```
Comment (edherm) [inline public/javascripts/task.js:106]: Nice work moving the logic for table cell and delete button into their own methods!

File: routes/home.js [hunk 3]
```diff
@@ -61,8 +61,8 @@ router.post(
 
     const newGroup = await db.Group.create({ name });
     await db.UserGroup.create({ userId, groupId: newGroup.id });
-    // res.redirect("/home");
-    res.send(newGroup);
+    res.redirect("/home");
+    // res.send(newGroup);
```
Comment (edherm) [inline routes/home.js]: If you aren't using this line of code, remove it for cleanliness.

**Review by edherm — CHANGES_REQUESTED** [review]
Just a few small things for you to look over here. Let me know when you've address all of my comments and I will come back to approve/merge.

File: routes/home.js (hunk 3 above)
Comment (jon-wehner) [reply routes/home.js]: Line removed

File: public/javascripts/group.js (hunk 1 above)
Comment (jon-wehner) [reply public/javascripts/group.js]: Removed

**Review by edherm — APPROVED** [review]
Nice work, Jon!

**Pull request merged**